lego-webapp/app/routes/hiddenAdmin/components/BannerOverview.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Card,
+  ConfirmModal,
+  Icon,
+  LinkButton,
+  Page,
+} from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { Trash2 } from 'lucide-react';
+import { ContentSection, ContentMain } from '~/components/Content';
+import { deleteBanner, fetchAllBanners } from '~/redux/actions/BannerActions';
+import { useAppDispatch, useAppSelector } from '~/redux/hooks';
+import { selectAllBanners } from '~/redux/slices/banner';
+
+const BannerOverview = () => {
+  const dispatch = useAppDispatch();
+  usePreparedEffect('fetchAllBanners', () => dispatch(fetchAllBanners()), []);
+  const allBanners = useAppSelector((state) => selectAllBanners(state));
+  return (
+    <Page
+      title={'Banner List'}
```

Comment:
```suggestion
      title="Bannere"
```

Is there a better Norwegian word for this?